Lottery.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+public class Lottery {
+    public static ball getRandomBall(){
+        Random random = new Random();
+        int numb = random.nextInt(100);
+        ColorSupplier ColorsSupplier = new ColorSupplier();
+        Colors gege = ColorsSupplier.getRandomColor();

```

Comment:
The variable name `gege` is not descriptive. Consider renaming it to something more meaningful, like `randomColor`, to improve code readability.